src/lib/storage/dictionary_segment.hpp

```diff
@@ -20,7 +20,8 @@ template <typename T>
 class DictionarySegment : public BaseDictionarySegment {
  public:
   explicit DictionarySegment(const std::shared_ptr<const pmr_vector<T>>& dictionary,
-                             const std::shared_ptr<const BaseCompressedVector>& attribute_vector);
+                             const std::shared_ptr<const BaseCompressedVector>& attribute_vector,
+                             const bool segment_contains_null_values = false);
```

Comment:
Do we need this argument?

Reply:
I don't know what you mean. It's used to pass the information when the encoder create the segment.

Reply:
**default** argument, sorry

Reply:
Removed.